frontend/src/metabase-types/analytics/search.ts

```diff
@@ -0,0 +1,42 @@
+type SearchContentType =
```

Comment:
Enums are duplicated by design to force you to update the enum definition in the schema if something changes